noah/src/main/resources/crawler/promedmail.conf

```diff
@@ -16,27 +16,22 @@ crawljax.waitafterevent = 50
 #Disable inserting random data into forms
 crawljax.insertrandomdataforms = false
 
-#Dont Click Elements
+#Maximum depth is set to infinite
+crawljax.maxdepth = 0
+
+#Click Elements
 builder {
   crawlrules {
-    element {
-      div {
-        id = ["header"]
-        class = ["navbar", "announcements", "copyright"]
-      }
-      table {
-        class = ["region_title"]
-      }
-
-      span {
-        class = ["printable"]
+    clickelements {
+      a {
+        id = []
+        href = ["http://promedmail.org"]
+        class = ["lcl"]
+        text = ["« prev"]
```

Comment:
Yes.... It works to my knowledge. Would you like to see it work? It does not take much time.